src/renderer/components/Reactions/styles.module.scss

```diff
@@ -0,0 +1,44 @@
+.reactions {
+  cursor: pointer;
+  margin-right: 10px;
+  position: relative;
+  top: 20px;
+}
+
+.emoji {
+  background-color: var(--messageIncomingBg);
+  border-radius: 24px;
+  border: solid 1px rgba(100, 100, 100, 0.5);
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
```

Comment:
the border is still a bit hard,  cmp. https://github.com/deltachat/deltachat-desktop/pull/2964#issuecomment-1899250677 , 0.3 alpha fixes that

```suggestion
  border: solid 1px rgba(100, 100, 100, 0.3);
```

in addition, i removed the box-shadow here completely. it affects only areas outside the bubble - and using the same, flat style as the normal bubbles just looks better and less 

before / after (including the font change):

![Screenshot 2024-01-22 at 21 03 48](https://github.com/deltachat/deltachat-desktop/assets/9800740/b47c4a7f-751d-4f3f-91f0-01024cbe7fc9) 
![Screenshot 2024-01-22 at 21 08 20](https://github.com/deltachat/deltachat-desktop/assets/9800740/85342ccd-a4a9-4c8b-8c56-52e0315fad3b)